packages/paymasters/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@opengsn/paymasters",
+  "license": "MIT",
+  "version": "2.2.2",
+  "scripts": {
+    "truffle-compile": "truffle compile --compile-all",
+    "typechain-generate": "yarn truffle-compile && typechain --target truffle-v5 './build/contracts/**/*.json'",
+    "tsc": "tsc",
+    "lint": "yarn run lint:js && yarn run lint:sol",
+    "lint:js": "eslint -f unix .",
+    "lint:sol": "solhint -f unix \"contracts/**/*.sol\" --max-warnings 0",
+    "test": "yarn tsc && yarn truffle-test-compile-all",
+    "truffle-test-compile-all": "truffle test --compile-all --network npmtest",
+    "rm-dist": "rm -rf tsconfig.tsbuildinfo dist build"
+  },
+  "files": [
+    "src/*",
+    "types/*",
+    "contracts/*",
+    "build/contracts/*",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^3.4.0-solc-0.7",
+    "@opengsn/dev": "^2.2.2",
+    "@opengsn/common": "^2.2.2",
+    "@opengsn/provider": "^2.2.2",
+    "@opengsn/contracts": "^2.2.2",
+    "@typechain/truffle-v5": "^4.0.1",
```

Comment:
I think types are only dev-dependencies. we should "export" such deps (general issue, not only this package.

Reply:
Moved `@typechain/truffle-v5` to `devDependencies` as it should not be referenced AFAIK.